client/blocks/continue-as-user/index.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function ContinueAsUser( { userName, redirectUrl } ) {
+	const translate = useTranslate();
+
+	if ( ! userName || ! redirectUrl ) {
+		return null;
+	}
+
+	const redirectLink = <a href={ redirectUrl }>{ userName }</a>;
+
+	return (
+		<div className="continue-as-user">
+			{ translate( 'or continue as {{redirectLink/}}', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
You're right! Good bot 🤖